packages/node-build-scripts/generate-css-variables.js

```diff
@@ -52,7 +52,7 @@ if (!args["retainDefault"]) {
 variablesScss = COPYRIGHT_HEADER + "\n" + variablesScss;
 
 if (!fs.existsSync(`${DEST_DIR}/scss`)) {
-    fs.mkdirSync(`${DEST_DIR}/scss`);
+    fs.mkdirSync(`${DEST_DIR}/scss`, { recursive: true });
```

Comment:
interesting; do you remember why you made this change?

Reply:
This was to address the circle build was failing during `yarn compile`. i think because this has to run in the compile stage instead of the `dist` stage like it does for `packages/core`, it must have been running into this line before `compile:css` is able to create the `lib` directory

Reply:
ok cool, that checks out 👍🏽